crates/bevy_render/src/mesh/morph/visitors.rs

```diff
@@ -0,0 +1,141 @@
+use bevy_core::{Pod, Zeroable};
+use bevy_math::Vec3;
+use wgpu::VertexFormat;
+
+use crate::mesh::{Mesh, MeshVertexAttribute};
+
+/// The set of attributes conviniently accessible in the `gltf` crate.
+///
+/// Also the only ones useable with the current morph targets implementation.
+/// See `morph.wgsl`.
+pub const HARDCODED_ATTRIBUTES: &[MeshVertexAttribute] = &[
+    Mesh::ATTRIBUTE_POSITION,
+    Mesh::ATTRIBUTE_NORMAL,
+    MeshVertexAttribute::new("Vertex_Tangent_Morph_Attribute", 7, VertexFormat::Float32x3),
```

Comment:
Could this one be added as a const at the same place at the other attributes? https://github.com/bevyengine/bevy/blob/a35ed552fafaffce8a28c7d47b3e294d755ff096/crates/bevy_render/src/mesh/mesh/mod.rs#L58-L84

Reply:
Ooops, sorry for the late reply. I can do this, however, `Vertex_Tangent_Morph_Attribute` doesn't exist per-se. In fact, I don't really use this, apart to return an error that can't possibly occur in `MorphTargetImage::new`, so I removed it. The idea was to provide an upgrade path for when we support arbitrary vertex attribute animation. Though I now recall my philosophy of never writing code you don't use, so boop, I remove it. Replacing it with an associated constant of `MorphAttributes`.